contact.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>Contact Page</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>
+		
+	<body>
+		
+		<nav class="nav_bar">
+			<p>Karen Schulz</p>
+			
+			<div>
+				<a href="portfolio_home.html">Home</a>
+				<a href="portfolio.html">Portfolio</a>
+			</div>
+			
+		</nav>
+		
+		<article>
+			<div class="contact"><img class="picture-of-me" src="images/half_body.png" alt="my_profile_picture">
+		<br>
+			<p>Don't be a stranger and drop me a line!</p>
+			
+			<a href=mailto:sirk.schulz@gmail.com>Email Me</a>
+		
+		<br>
+			<p>Otherwise you can submit your contact details and I will be in touch:</p>
+		
+		
+		<form>
+			
+			<p>
+				<label>Where did you hear about me?</label>
+			</p>
+			
+			<p>
+				<label for="friend"><h6>Friend</h6></label>
+				<input type="radio" name="my-radio" id="friend" value="friend" />
+			</p>
+			
+			<p>
+				<label for="githut"><h6>Github</h6></label>
+				<input type="radio" name="my-radio" id="github" value="ngithub" />
+			</p>
+			
+			<p>
+				<label for="my-checkbox">Would you like me to send you my portfolio?</label>
+				<input type="checkbox" name="my-checkbox" value="checkbox" />
+			</p>
+			
+			<p>
+				<label for="email"><h6>Please enter your email:</h6></label>
+				<input type="email"/>
+				<input type="submit" value="submit"/>
+			</p>
```

Comment:
i am not sure if p tags should be using inside of a form. I believe you should use divs here instead as these are not paragraphs

Reply:
I asked Rayna and she said it is possible to use<p> inside a form. You cannot use <p> whit <h1> for example. So I kept the <p> tags